src/sql/workbench/services/notebook/browser/models/cell.ts

```diff
@@ -349,7 +349,10 @@ export class CellModel extends Disposable implements ICellModel {
 	private attachImageFromSource(newSource: string | string[]): string | string[] {
 		if (!Array.isArray(newSource) && this.isValidBase64OctetStream(newSource)) {
 			let results;
-			while ((results = validBase64OctetStreamRegex.exec(newSource)) !== null) {
+			// only replace the base64 value if it's from markdown [](base64value) not html tags <img src="base64value">
+			let validImageTag = /<img\s+[^>]*src="([^"]*)"[^>]*>/;
```

Comment:
It would be a malformed tag if there is a > inside the elements attributes.

Reply:
Not necessarily - e.g. [https://github.com/Microsoft/azuredatastudio/issues/new?title=Test<>Test](https://github.com/Microsoft/azuredatastudio/issues/new?title=Test<>Test) is a valid URL, so you could have an image with something like that as a URL.

Reply:
The link was just an example URL since I don't have an image available that has a <> in the URL - but for example an image with a URL such as `https://test.com/myimage.png?somedata=Test<>Test` would be possible for someone to enter. 

So if I paste this into the text cell : `<img src="https://test.com/myimage.png?somedata=Test<>Test" >` is that URL going to be escaped before it gets to this logic?

Granted this is getting into edge case territory, but we need to make sure we're at least aware of these issues and trying to handle them if we can (and if not then making it clear that that's intended behavior for now)

Reply:
Here's an actual URL you can use for testing : [https://raw.githubusercontent.com/microsoft/azuredatastudio/main/resources/win32/code_150x150.png?somedata=<>](https://raw.githubusercontent.com/microsoft/azuredatastudio/main/resources/win32/code_150x150.png?somedata=<>)

Reply:
For the scenario am trying to fix, it's very specific to copy paste images and the src value will always be a valid base64 octet value.

Reply:
Yeah, good point. Even though the regex would get the incorrect value if it has a > in it that shouldn't matter. It would be a good idea to call that out specifically in a comment though so later on we don't try to re-use this for something that does actually care about handling absolute URLs.